collectors/VMPropertiesCollector.py

```diff
@@ -0,0 +1,84 @@
+from BaseCollector import BaseCollector
+import os, time, json
+from prometheus_client.core import GaugeMetricFamily
+from tools.Resources import Resources
+from tools.YamlRead import YamlRead
+
+
+class VMPropertiesCollector(BaseCollector):
+    def __init__(self):
+        self.wait_for_inventory_data()
+        self.property_yaml = YamlRead('collectors/property.yaml').run()
+        self.g = GaugeMetricFamily('vrops_vm_properties', 'testtest',
+                labels=['cluster', 'datacenter', 'virtualmachine', 'hostsystem', 'propkey'])
```

Comment:
please rename cluster to: `vccluster`
cluster is predefined and shouldn't be used